docs/quick-start.md

```diff
@@ -0,0 +1,84 @@
+---
+hide:
+  - toc
+---
+# Getting started with CodeSee
+
+An overview to the primary features of CodeSee, including recordings, data flows, stories, and sharing.
```

Comment:
This sentence is a little hard to follow. I think it might be a little more clear with something like: "This page is an overview..." or "Let's start with an overview..."